lib/spack/spack/cmd/list.py

```diff
@@ -35,7 +35,10 @@
 import spack
 from llnl.util.tty.colify import colify
 
-description = "print available spack packages to stdout in different formats"
+description = "query and show packages that can be installed"
```

Comment:
Based on my comment above, it might be better to say "list available packages". But I get that you want to make it clear that `spack list` can also accept an argument for searching.